### PR TITLE
Release 3.6.0

### PR DESCRIPTION
**Pull request by joaocsilva**

### CODE REVIEW

File: includes/phing/props/drone.props [hunk 1]
```diff
@@ -9,3 +9,5 @@ behat.wd_host.url = http://selenium:4444/wd/hub
 share.path = /cache
 project.db.file = ./dump.sql
 project.url.base = http://web:8080
+db.dl.username =
+db.dl.password =
```
Comment (jonhy81) [inline includes/phing/props/drone.props]: Why are you doing a reset to values here?

**Review by jonhy81 — CHANGES_REQUESTED** [review]
Not clear to me why you are doing a reset to values for drone.

File: includes/phing/props/drone.props (hunk 1 above)
Comment (joaocsilva) [reply includes/phing/props/drone.props]: Is just to make sure that the variables are empty, to use the values from the ENV vars.
But if it is not needed I can remove, let me know.

Comment (joaocsilva) [reply includes/phing/props/drone.props]: Ok, I'll remove those variables and let the project decide if leave empty or not.

Comment (joaocsilva) [inline includes/phing/props/drone.props]:
```suggestion
```

**Review by zarabatana — APPROVED** [review]

**Pull request merged**